source/frontend/src/features/properties/map/acquisition/add/AddAcquisitionForm.tsx

```diff
@@ -204,6 +205,7 @@ export const AddAcquisitionForm = React.forwardRef<
                 <ContactInputContainer
                   field="ownerSolicitor.contact"
                   View={ContactInputView}
+                  restrictContactType={RestrictContactType.ONLY_ORGANIZATIONS}
```

Comment:
thanks for doing this. could you please add the restriction to only persons to owner representative below in this form?